frontend/src/pages/AbsencePlanner.tsx

```diff
@@ -21,34 +21,49 @@ import {
   getUsersInGroups,
   getGroups,
 } from "../utils";
-import {
-  eachDayOfInterval,
-  Interval,
-  format as formatDate,
-  addDays,
-} from "date-fns";
+import { eachDayOfInterval, Interval, format as formatDate } from "date-fns";
 import LoadingOverlay from "react-loading-overlay-ts";
 import ClimbingBoxLoader from "react-spinners/ClimbingBoxLoader";
 import { HeaderUser } from "../components/HeaderUser";
 import { Chart } from "react-google-charts";
+import trash from "../icons/trash.svg";
+import pencil from "../icons/pencil.svg";
 
-export const VacationPlanner = () => {
+export const AbsencePlanner = () => {
   const [startDate, setStartDate] = useState<Date>(undefined);
   const [endDate, setEndDate] = useState<Date>(undefined);
   const [toastList, setToastList] = useState<ToastMsg[]>([]);
-  const [vacationRows, setVacationRows] = useState<[]>([]);
+  const [absenceRows, setAbsenceRows] = useState<[]>([]);
   const [selectedGroup, setSelectedGroup] = useState<{
     id: number;
     name: string;
   }>(undefined);
   const [redmineGroups, setRedmineGroups] = useState<[]>([]);
 
   const [isLoading, setIsLoading] = useState(false);
+  const [tableData, setTableData] = useState<
+    { startDate: Date; endDate: Date; userName: string; entryIds: number[] }[]
+  >([]);
+  const absenceFrom: Date = getAbsenceFrom();
+  const absenceTo: Date = getAbsenceTo();
+  const [reloadPage, setReloadPage] = useState<boolean>(false);
+
+  function getAbsenceFrom() {
+    let today = new Date();
+
+    return new Date(today.setMonth(today.getMonth() - 1));
+  }
+
+  function getAbsenceTo() {
+    let today = new Date();
+
+    return new Date(today.setMonth(today.getMonth() + 12));
+  }
```

Comment:
In the rest of the file you use arrow functions (as we do in the rest of the project as well). Is there any reason you did it different here? If no, I'd convert them into arrow functions as well to keep it consistent and prevent confusion.